src/views/UserProfileView/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import ProfileComponent from '../../components/Profile';
+import { profileData } from '../../redux/actions/userProfileActions/actions';
+
+class ProfileView extends Component {
+  state = { };
+
+  componentDidMount() {
+    const user = JSON.parse(localStorage.getItem('user'));
+    const payload = {};
+    const { username } = user;
+    payload.username = username;
+    const { profileData: profileAction } = this.props;
+    profileAction(payload);
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
Why is this empty?

Reply:
Hello,
This is extinct. I found a workaround for the implementation I had intended for it.
I have expunged it from the code base.
Thank you.